### PR TITLE
add pytools,pymbolic to dependencies

### PR DESCRIPTION
**Pull request by matthiasdiener**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ $ spack install pocl
 
 ```bash
 $ pip install pyvisfile
-$ for m in dagrt leap loopy meshmode grudge mirgecom; do cd $m && pip install -e . && cd ..; done
+$ for m in dagrt leap loopy meshmode grudge pytools mirgecom; do cd $m && pip install -e . && cd ..; done
```
Comment (inducer) [inline README.md:55]: `pytools` should be installed before everything else. Otherwise the other stuff will have pulled it in as a dependency already.

Comment (matthiasdiener) [reply README.md]: Should be fixed now.

**Comment by inducer** [comment]
Actually, maybe add pymbolic while you're at it.

**Comment by matthiasdiener** [comment]
> Actually, maybe add pymbolic while you're at it.

Ok, added.

**Pull request merged**